src/Compilers/Server/VBCSCompilerTests/CompilerServerTests.cs

```diff
@@ -256,16 +256,16 @@ public async Task ServerFailsWithLongTempPathUnix()
                 new[] { new KeyValuePair<string, string>("TMPDIR", newTempDir.Path) },
                 async () =>
             {
-                using (var serverData = ServerUtil.CreateServer())
+                using (var serverData = await ServerUtil.CreateServer())
                 {
                     var result = RunCommandLineCompiler(
                         CSharpCompilerClientExecutable,
                         $"/shared:{serverData.PipeName} /nologo hello.cs",
                         _tempDirectory,
                         s_helloWorldSrcCs,
-                        shouldRunOnServer: false);
+                        shouldRunOnServer: true);
```

Comment:
Can you explain the false->true and 0->1 changes here?

Reply:
The test previously tested that the server failed. So the server would start, then shutdown, and the command line compiler would fall back to in-memory compilation. So we previously asserted that the request would not run on the server, and that the server exited without processing any compilations.

Now we do the opposite (because I fixed it) -- the compilation runs on the server successfully, and the server processes one compilation.